### PR TITLE
[Sprint 2][Team 1][Task 2] Implement logging for /users/{user_id}

### LINKED ISSUE #67: [SPRINT 2][TEAM 1][TASK 2] Implement logging for /users/{user_id}
**Is your feature request related to a problem? Please describe.**
Project requires clear, descriptive logging for the /users/{user_id} API

**Describe the solution you'd like**
Appropriate logging for calls to /users/{user_id} 

**Describe alternatives you've considered**
N/A - feature is required as-is to function with rest of team/class

### PR DESCRIPTION
**Pull request by philckim**
added console logging to init.py files for {userId} methods

### CODE REVIEW

**Comment by Raotrux** [comment]
The logging itself looks good but where are all of these other changes coming from? I have no idea what's going on with most of that, it seems like other people's tasks, so I'm hesitant to approve.

File: automation/functions/script.ps1 [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+    [Parameter(Mandatory = $True)]
+    [string]
+    $location
+)
+
 #sign into azure account 
 Connect-AzAccount
 
+#Will change this to NSC raw link after merge
+$templateUri = "https://raw.githubusercontent.com/selinapn/ad440-winter2021-tuesday-repo/automationspn-s2/automation/functions/azuredeploy.json"
```
Comment (RealDogDad) [inline automation/functions/script.ps1]: Is this handled by Deploy team?

**Review by RealDogDad — COMMENTED** [review]
Just wondering if Team Deploy is to handle the URIs for Azure Resouces. Otherwise you did what I did. Although maybe it would be smart to set a logging configuration somewhere. I used a python dict for a config,.

**Review by RealDogDad — APPROVED** [review]
Should we add a dict for logging configuration for easy changes later?

**Review by jlsnow301 — CHANGES_REQUESTED** [review]
Looks like some merge artefacts going on. Please merge in upstream dev again, some of these files shouldn't be here

**Review by jlsnow301 — APPROVED** [review]
ok great

**Pull request merged**